modules/virt-viewing-network-state-of-node-console.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * networking/k8s_nmstate/k8s-observing-node-network-state.adoc
+
+:_content-type: PROCEDURE
+[id="virt-viewing-network-state-of-node-console_{context}"]
+= Viewing the network state of a node from the web console
+
+As an administrator, you can use the {product-title} web console to observe `NodeNetworkState` resources and network interfaces, and access network details.
+
+.Procedure
+. Navigate to *Networking* → *NodeNetworkState*.
++
+In the *NodeNetworkState* page, you can view the list of `NodeNetworkState` resources, and the interfaces that are created on the nodes.
+. Click on the `NodeNetworkState` resource name listed in the *Name* column to access the detailed information about NNS.
+. Click on the *>* icon to expand and view the *Network Details* section for the `NodeNetworkState` resource. Alternatively, you can click on each interface type under the *Network interface* column to view the network details.
+
+You can use *Filter* based on *Interface state*, *Interface type*, and *IP*, or the search bar based on criteria *Name* or *Label*, to narrow down the displayed `NodeNetworkState` resources.
```

Comment:
Should this be attached to maybe step 1 above?

Reply:
That's a nice suggestion!